components/colorslider/stories/template.js

```diff
@@ -0,0 +1,42 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { Template as ColorHandle } from "@spectrum-css/colorhandle/stories/template.js";
+
+
+import '../index.css';
+import '../skin.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorSlider",
+  customClasses = [],
+  isDisabled = false,
+  isFocused = false,
+  vertical,
+  colorHandleStyle = "--spectrum-picked-color: rgb(255, 0, 0);",
+  ...globals
+}) => {
+  return html`
+    <div class=${classMap({
+      [rootClass]: true,
+      'spectrum-ColorSlider--vertical': vertical,
+      'is-disabled': isDisabled,
+      'is-focused': isFocused,
+      ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+  })}
+  ${isDisabled}
+  ${isFocused}>
```

Comment:
What are these doing?

Reply:
I suspect these can be removed since the classes are being added above.

Reply:
hmm yeah you both are right they can be removed. 
I was basing this off of how we implemented some of the button components but I can see they are not needed here.